pyproject.toml

```diff
@@ -25,32 +26,32 @@ classifiers = [
     "Operating System :: Unix",
     "Operating System :: MacOS",
 ]
-requires-python = ">=3.9"
-dependencies = [
-    "aws-cdk-lib==2.48.0",
-    "aws-cdk.aws-lambda-python-alpha",
-    "constructs>=10.0.0,<11.0.0",
-    "boto3",
-    "opensearch-py",
-    "requests",
-    "python-jose"
-]
 
-[project.optional-dependencies]
-dev = [
-    "pytest==6.2.5",
-    "moto"
-]
+[tool.poetry.dependencies]
+python = ">=3.9,<4"
+# Required for aws-cdk.aws-lambda-python-alpha
+aws-cdk-lib = ">=2.66.0,<3.0.0"
```

Comment:
This is the only change to the dependencies - when running, I got an error message about the version of aws-cdk-lib, so I bumped it from `2.48.0` to `2.66.0`. If this causes problems, I can downgrade again, but `aws-cdk.aws-lambda-python-alpha` requires at least this version number.

Reply:
:+1: Good catch!